CHANGELOG.md

```diff
@@ -1,9 +1,17 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Command Generate - Add logs to explain what is going on.
+
+### Changed
+- Command Generate - Camelize all field names.
+- Command Generate - Camelize all reference names and remove `id` or `uuid` from name.
+
 ### Fixed
 - Command Generate - Fix package name.
 - Command Generate - Remove trailing spaces in `.env`.
+- Command Generate - Do not duplicate `createAt` and `updatedAt` if not named as camelCase in database.
```

Comment:
`createAt` => `createdAt`.
Do you take the time to review your own PR?
Such typo should be detected by ourself, not your reviewer.